x-pack/plugin/ent-search/licenses/commons-lang3-LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Licenses were duplicated where possible from the codebase, or retrieved in the case of json-schema-validator to ensure the license check is passed.